stdlib/format.mli

```diff
@@ -235,6 +235,14 @@ val pp_print_string : formatter -> string -> unit
 val print_string : string -> unit
 (** [pp_print_string ppf s] prints [s] in the current pretty-printing box. *)
 
+val pp_print_substring : first:int -> len:int -> formatter -> string -> unit
+val print_substring : first:int -> len:int -> string -> unit
+(** [pp_print_substring ~first ~len ppf s] prints the substring of [s]
+  that starts at index [start] and stop at index [start+len] in the current
+  pretty-printing box.
+  @since 5.1
+*)
```

Comment:
The docstring does not match the labels: `start` <-> `first`.

Personally, I didn't understand from the discussion why we don't use `pos` for the first label as in `StringLabels`. It seems needlessly noisy to introduce yet another convention of how to refer to the starting offset of a substring.

Reply:
I don't think that we should be beholden to the label choice of `StringLabels` outside of the labelled variant of the standard library and `pos` is an uninformative and ambiguous label name.

But if you prefer I could use it; after all `Format` API is already barely usable without `Fmt` due to its extremely verbose naming convention.

Reply:
I don't have a strong opinion, but for me `pos` means "position" and I have no doubt that it means the position of the (start of the) substring, and `len` its len. So i am fine with either `pos` and `first`.

Reply:
Note that `String` attributes a particular meaning to the term position (versus index, see the `String` module preamble). If `pos` is used one should check the function works when it is equal to `String.length s`.

Reply:
I don't have a strong opinion on this issue, so I'm happy to let @Octachron decide. In any case, the docstring should be corrected.